vcbuild.bat

```diff
@@ -28,15 +26,13 @@ set licensertf=
 set jslint=
 set buildnodeweak=
 set noetw=
-set noetw_arg=
 set noetw_msi_arg=
 set noperfctr=
-set noperfctr_arg=
 set noperfctr_msi_arg=
 set i18n_arg=
-set download_arg=
```

Comment:
There are multiple places where we override download_arg based on command line arguments. Maybe keep this one around instead of adding it to config_flags?